src/module/helpers/tokenUtils.mjs

```diff
@@ -0,0 +1,43 @@
+/**
+ * @param {DrawSteelActor} actor
+ * @param input
+ * @param fill
+ * @returns {Promise<void>}
+ */
+export async function showFloatyText(actor, input, fill) {
+  if (!canvas.scene) {
+    return;
+  }
+
+  const [token] = actor.getActiveTokens();
+
+  if (token && (typeof input === "number")) {
+    const gridSize = canvas.scene.grid.size;
+    const scrollingTextArgs = [
+      { x: token.x + gridSize / 2, y: token.y + gridSize - 20 },
```

Comment:
To get the center position of a token, simply use `token.center`. This current approach ignores token sizes.